src/config.ts

```diff
@@ -31,6 +38,8 @@ export default {
   SECURITY_MODE_JWT,
   SECURITY_MODE_STANDALONE,
 
+  cookieSecret: process.env.COOKIE_SECRET || cookieSecret,
```

Comment:
I would move this key under `hydra` and also prefix it `HYDRA_COOKIE_SECRET`

Reply:
Agreed